docs/xgc.md

```diff
@@ -44,7 +44,10 @@ Options that change the behavior of the Garbage Collector (GC).
 | [`preferredHeapBase`           ](#preferredheapbase           ) | Sets a memory range for the Java&trade; heap. (AIX&reg;, Linux&trade;, and Windows&trade; only) |
 | [`scvNoAdaptiveTenure`         ](#scvnoadaptivetenure         ) | Turns off the adaptive tenure age in the generational concurrent GC policy.                             |
 | [`scvTenureAge`                ](#scvtenureage                ) | Sets the initial scavenger tenure age in the generational concurrent GC policy.                         |
-| [`verboseFormat`               ](#verboseformat               ) | Sets the verbose GC format.                                                                             |
+| [`tlhIncrementSize`            ](#tlhincrementsize            ) | Sets the size of the thread local heap (TLH)  increment                                      |
```

Comment:
you can add:....  by which TLH will grow on every TLH refresh.

Reply:
Detail is in the section linked to (to keep the table simple)